docs/team/scarletblanks.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default.md
+title: "Peng Tat's Project Portfolio Page"
+---
+
+# Project: JABPRO
+## Overview JABPro aims to solve the problem of HR managers having to sort through tons of job applications.
```

Comment:
Tiny detail but could you move "JABPro aims to solve..." to a new line below ## Overview?